sherlock-platform/customization/src/com/google/sherlock/onboardingFlow/NewProjectStep.java

```diff
@@ -0,0 +1,28 @@
+/***
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***/
+
+package com.google.sherlock.onboardingFlow;
+
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import org.jetbrains.annotations.NotNull;
+
+//This is placeholder, more like an exploration and will be updated based on the UI design for onboarding
+public class NewProjectStep extends AnAction {
+  @Override
+  public void actionPerformed(@NotNull AnActionEvent e) {
+
```

Comment:
As we think about what to do with this button, we might want to consider a couple of options:
1.  Reuse the new project wizard. Might be redundant as we won't need most of it, but it has a nice multi-page (next -> next -> finish) capability
2. Create a simple single-dialog UI to ask the user for project directory + project name, then create a project of type `EmptyProjectType` in that directory (not sure but we might need to create an empty module too).

I'm thinking of the latter. 

This can be in a follow up PR.  Can we keep #21 open until we finish this work (or split it and close half of it)?

Reply:
Created a follow up issue #24